packages/appstore/src/components/total-assets/total-assets.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Popover, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { formatMoney } from '@deriv/shared';
+
+type TTotalAssets = {
+    amount: string;
+    currency: string;
+    category: 'real' | 'demo';
+};
+
+const TotalAssets = ({ amount, currency, category }: TTotalAssets) => {
+    return (
+        <div className='total-assets'>
+            <Text size='sm' weight='bold' className='total-assets-amount'>
```

Comment:
There is no `sm` size.
also default value for text size is `s` so you can simply remove it.